remake.yml

```diff
@@ -28,7 +29,7 @@ targets:
 
   # Configuration
   states:
-    command: c(I(c('WI','MN','MI','IL','IN', 'IA', 'OH', 'NY')))
+    command: c(I(c('AL','AZ','AR','CA','CO','CT','DE','DC','FL','GA','ID','IL','IN','IA','KS','KY','LA','ME','MD','MA','MI','MN','MS','MO','MT','NE','NV','NH','NJ','NM','NY','NC','ND','OH','OK','OR','PA','RI','SC','SD','TN','TX','UT','VT','VA','WA','WV','WI','WY','AK','HI','GU','PR')))
```

Comment:
Perhaps you just prefer long lines and quicker scrolling, but do you know that you _can_ use line breaks within these `command` blocks? Not every break location works - I think you need the first line break to happen right after an `(` - but this formatting is accepted by **scipiper**:
```yml
  states:
    command: c(I(c(
      'AL','AZ','AR','CA','CO','CT','DE','DC','FL','GA','ID','IL','IN','IA',
      'KS','KY','LA','ME','MD','MA','MI','MN','MS','MO','MT','NE','NV','NH',
      'NJ','NM','NY','NC','ND','OH','OK','OR','PA','RI','SC','SD','TN','TX',
      'UT','VT','VA','WA','WV','WI','WY','AK','HI','GU','PR')))
```

Reply:
Huh - I had some in there, but must have had the first one placed incorrectly, since it was throwing an error. Good to know!